### PR TITLE
Add test case in update operation

### PR DESCRIPTION
**Pull request by leonwanghui**
Signed-off-by: leonwanghui <wanghui71leon@gmail.com>

This patch is first part of #66 , focusing on update scenario.

### CODE REVIEW

File: 2.13/tests/test/test.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ function testLifecycle (testCase) {
             validBody.plan_id = currentPlanId
           }
           if (testCase.parameters) {
```
Comment (zhongyi-zhang) [inline 2.13/tests/test/test.js]: the condition should also be fixed.

File: 2.13/mocks/mockOSB.js [hunk 2]
```diff
@@ -109,6 +109,22 @@ app.patch('/v2/service_instances/:instance_id', function (req, res) {
...
+  }
+
+  if (req.body.plan_id !== '' && serviceInstanceExists(req.body.service_id, req.body.plan_id, req.params.instance_id)) {
+    res.status(200).send({})
+    return
+  } else {
+    var found = findWhichContains(serviceInstances, 'instance_id', req.params.instance_id)
+    var i = serviceInstances.indexOf(found)
+    if (i > -1) {
+      serviceInstances[i].planId = req.body.planId
```
Comment (zhongyi-zhang) [inline 2.13/mocks/mockOSB.js]: It is `plan_id ` in request body. I think that's why CI failed.

Comment (leonwanghui) [reply 2.13/mocks/mockOSB.js]: Got it

**Comment by leonwanghui** [comment]
@norshtein Please take a review, thanks!

**Pull request merged**